src/ci-test-compile.sh

```diff
@@ -86,38 +86,38 @@ run_build() {
 }
```

Comment:
The CI stuff isn't part of the BabelStream source - can this live in the top level directory?